### PR TITLE
feat(board): add ability to add and view cards

### LINKED ISSUE #40: As a participant, I can create a Card.

### PR DESCRIPTION
**Pull request by benox3**
Closes #40

### CODE REVIEW

File: src/components/Board/BoardContainer.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import Board from './Board';
 import useGroup from '../../lib/hooks/useGroup';
+import useCards from '../../lib/hooks/useCards';
 import { Redirect } from 'react-router-dom';
+import useAddCard from '../../lib/hooks/useAddCard';
```
Comment (remarkablemark) [inline src/components/Board/BoardContainer.tsx]: Minor: we could probably set up some eslint sort import rule here

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "react-firebase-hooks": "^2.1.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
+    "storybook": "^5.3.0",
```
Comment (remarkablemark) [inline package.json]: Actually, do you mind reverting or dropping this commit? I opened #42 that upgrades all dependencies

File: src/components/Board/BoardContainer.tsx (hunk 1 above)
Comment (remarkablemark) [reply src/components/Board/BoardContainer.tsx]: See #43

File: package.json (hunk 2 above)
Comment (benox3) [reply package.json]: Let's get yours merged in first and  then I'll fix the conflict accordingly

Comment (remarkablemark) [reply package.json]: Sounds good 👍

**Comment by remarkablemark** [comment]
Please merge/rebase `master` when you get a chance

**Pull request merged**